docs/path-for-tracked-resource-types.md

```diff
@@ -0,0 +1,33 @@
+# PathForTrackedResourceTypes
+
+## Category
+
+ARM Error
+
+## Applies to
+
+ARM OpenAPI(swagger) specs
+
+## Related ARM Guideline Code
+
+- RPC-Put-V1-01, RPC-Get-V1-11  
+
+## Output Message
+
+The path must be under a subscription and resource group for tracked resource types.
+
+## Description
+
+For a operation, If the URI segment has a subscription, then the URI needs to have a resource group segment as well.
+
```

Comment:
A tracked resource is supported only under a resource group scope, so all API paths for a tracked resource must contain a subscriptions and resourceGroups segement